client/src/common/components/Members/components/MemberButton.jsx

```diff
@@ -1,22 +1,41 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 
-const MemberButton = ({ onDisplayDetails, member, popperRef }) => (
-  <button
-    className="member-button"
-    ref={popperRef}
-    title={member.displayName}
-    type="button"
-  >
-    <img
-      alt={`${member.displayName} avatar`}
-      className="member-button__avatar"
-      onClick={onDisplayDetails}
-      src={member.avatarUrl}
-      title={member.displayName}
-    />
-  </button>
-);
+import UserIconPlaceholder from 'assets/images/user.svg';
```

Comment:
I'm not sure, but if we have to import images we should use '../../assets/images/user.svg' now.

Reply:
@marekrozmus   Should it be a full relative path?